### PR TITLE
Update CHANGELOG to mention deprecation of `file()` for `files()`.

### PR DESCRIPTION
**Pull request by essiene**
I missed this update in #4334.

Issue: #4122

# Checklist

If applicable:
- [x] I have updated `CHANGELOG.md`
- [ ] I have updated the documentation (README.md, docs/, demos/)
- [ ] I have updated the config schema (cli/src/config-schema.json)
- [ ] I have added tests to cover my changes

### CODE REVIEW

**Review by martinvonz — APPROVED** [review]
Thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -94,8 +94,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
   commit to commit. It now either follows the flags `--edit|--no-edit` or it
   gets the mode from `ui.movement.edit`.
 
-* `conflict` revset has been renamed to `conflicts`. With this, all revsets that
-  could potentially return multiple results are named in the plural.
+* `conflict()` and `file()` revsets have been renamed to `conflicts()` and `files()`
+  respectively. With this, all revsets that could potentially return multiple
+  results are named in the plural.
```
Comment (yuja) [inline CHANGELOG.md:99]: You'll also need to move this to `[Unreleased]: Deprecations`. Can you send a follow up?

Comment (essiene) [reply CHANGELOG.md:99]: Whoops. I forgot to move this after the last release. Sending a follow up now. Thanks for catching!

Comment (essiene) [reply CHANGELOG.md:99]: Sent #4532.

**Pull request merged**